servers/account-data-deleter/src/queueHandlers/exportStateHandler.ts

```diff
@@ -48,13 +49,20 @@ export class ExportStateHandler extends QueuePoller<{ Message: string }> {
    * (underlying call to AccountDeleteDataService completed without error)
    */
   async handleMessage(message: { Message: string }) {
+    let body: ExportRequested | ExportPartComplete;
+
     try {
-      let body: ExportRequested | ExportPartComplete;
       // message is nested in layers of SNS/SQS overhead
       if ('Message' in message && message.Message != null) {
         body = JSON.parse(message.Message);
       } else {
-        throw new Error('Invalid message body');
+        const error = new Error('Invalid SQS message body');
+        Sentry.addBreadcrumb({
+          message: 'Invalid SQS message body',
+          data: { messageBody: message },
+        });
+        Sentry.captureException(error);
```

Comment:
is there a reason you wouldn't want to capture the exception in the `catch` block below instead?

related - i'm not sure how the breadcrumb in the `catch` block will show up if the exception is already captured here...

Reply:
good catch, this should be captured in the outer try/catch block